beetmoverscript/templates/thunderbird_candidates.yml

```diff
@@ -104,6 +104,10 @@ mapping:
       s3_key: beetmover-checksums/source/thunderbird-{{ version }}.checksums.asc
       destinations:
         - beetmover-checksums/source/thunderbird-{{ version }}.checksums.asc
+    buildhub.json:
+      s3_key: {{ platform }}/{{ locale }}.buildhub.json
+      destinations:
+        - {{ platform }}/{{ locale }}.buildhub.json
```

Comment:
I assume you meant `{{ platform }}/{{ locale }}.buildhub.json` here instead.

Reply:
this is what I have, no?